packages/passwordless-auth-construct/API.md

```diff
@@ -6,30 +6,45 @@
 

```

Comment:
These are breaking changes that will be disruptive to preview customers. Can we take an additive approach here and deprecate the fields as part of the GA release?